python_modules/libraries/dagster-wandb/setup.py

```diff
@@ -35,7 +35,7 @@ def get_version() -> str:
     python_requires=">=3.8,<3.13",
     install_requires=[
         f"dagster{pin}",
-        "wandb>=0.13.5,<0.15.5",
+        "wandb>=0.13.5,<1.0",
```

Comment:
@chrishiste does the lower bound need to be bumped to  `0.15.11` since thats when the `wandb.sdk.launch.launch_add` imports were moved around in 

https://github.com/wandb/wandb/commit/5fca6eeeae478bb9d4265b16394c1428bb6330e2
?

Reply:
Yes sir, great catch

Reply:
ok incorporated in to https://github.com/dagster-io/dagster/pull/20135 as well

Reply:
Thanks I'm closing this one